signalfx_lambda/utils.py

```diff
@@ -54,7 +54,7 @@ def get_metrics_url():
         url = os.environ.get('SIGNALFX_METRICS_URL')
 
     if not url:
-        url = os.environ.get('SIGNALFX_ENDPOINT_URL', 'https://ingest.us0.signalfx.com')
+        url = os.environ.get('SIGNALFX_ENDPOINT_URL', 'https://pops.signalfx.com')
```

Comment:
last PI, we were specifically asked to remove all references to pops, as it is being deprecated (or perhaps already removed?). Is this a deliberate change?

Reply:
Not delibrate. I think my master copy was probably outdated.

Reply:
Fixed.